### ISSUE TITLE
Feature request for custom Snippet prompt

### ISSUE THREAD

**Issue by firede** [zeit-ui/react#324]
# Feature request 🚀

  - [ ] I will create Pull Request
  - [x] It's just a suggestion

### Expected

  - Snippet with custom prompt
  - Snippet without prompt

### Examples

```jsx
<Snippet {...props} prompt=">" />
```
<img src="https://user-images.githubusercontent.com/157338/87219096-1b0e1680-c38b-11ea-8cb0-a1c54da370fd.png" width="430" />

```jsx
<Snippet {...props} prompt="#" />
```
<img src="https://user-images.githubusercontent.com/157338/87219286-683eb800-c38c-11ea-96d0-5d90e24022df.png" width="324" />

```jsx
<Snippet {...props} prompt={false} />
```
<img src="https://user-images.githubusercontent.com/157338/87219231-0da55c00-c38c-11ea-8775-dce7c87c814c.png" width="324" />

**Comment by unix** [comment]
This feature has been added to the canary branch, you can find the usage in the [canary's document](https://react-rc.zeit-ui.co/en-us/components/snippet).
[release note](https://github.com/zeit-ui/react/releases/tag/v1.8.0-canary.0)

**Comment by firede** [comment]
Awesome! 👍 

But when I try this, I got an unnecessary space:
<img src="https://user-images.githubusercontent.com/157338/87220888-42b8ab00-c39a-11ea-8c21-7f51430d213e.png" width="672" />

**Comment by unix** [comment]
This seems to be an unexpected mistake, I'll fix it ASAP.

**Comment by unix** [comment]
@firede The issue you mentioned has been fixed, and the new canary version `v1.8.0-canary.5` is already 1able.

**Comment by firede** [comment]
LGTM, thanks!
